### PR TITLE
Fix version.txt file contents

### PR DESCRIPTION
**Pull request by schellap**

### CODE REVIEW

**Comment by schellap** [comment]
@eerhardt PTAL.

File: build_projects/dotnet-host-build/CompileTargets.cs [hunk 1]
```diff
@@ -258,7 +258,7 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
             var arch = IsWinx86 ? "x86" : "x64";
             
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
-            var hostNugetversion = hostVersion.LatestHostVersion;
+            var hostNugetversion = hostVersion.LatestHostVersion.ToString();
```
Comment (eerhardt) [inline build_projects/dotnet-host-build/CompileTargets.cs]: If this bug doesn't convince people to not use `var`, I don't know what will.

**Comment by eerhardt** [comment]
:shipit:

**Comment by eerhardt** [comment]
@schellap - it appears that there is other issues with this VerInfo struct.  We have been getting the following build warning/error:

```
F:\Agent\_work\12\s\build_projects\shared-build-targets-utils\Utils\HostVersion.cs(34,27): warning CS0114: 'HostVersion.VerInfo.ToString()' hides inherited member 'ValueType.ToString()'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
```

Fixing that would also solve this issue.

**Comment by eerhardt** [comment]
Fixing warning with #69

**Pull request merged**